### PR TITLE
Template for the onboarding questions

### PR DESCRIPTION
**Pull request by jorgeschnura**
Created a template for the onboarding questions.
@eiso please review if I've done it right
@tsolakoua please review as well if there's anything else you'd like to add

### CODE REVIEW

**Comment by tsolakoua** [comment]
@jorgeschnura you could also include: "Would you like to share with us a story about yourself?" 
and/or "What is your favorite quote?"

**Comment by tsolakoua** [comment]
@jorgeschnura I committed my questions in the file you created, hope that's ok.

**Comment by eiso** [comment]
@anna.tsolakou could you actually add a commit that links creating this PR from the onboarding checklist template

**Comment by tsolakoua** [comment]
@eiso I asked @dpordomingo to help me do that and it's not possible to create a PR with a link without having made manually before a specific branch for this PR (@dpordomingo not sure if I explain it properly....)

So what we decided is better to do is the following:
- create a README.md with some instructions (create a PR, edit the onboarding questions etc.)
- merge this PR
- add a link on the onboarding template to the README.md with all the instructions that the new member will follow.

**Comment by eiso** [comment]
Ah yes, you're right. Let's make those changes here then.

**Comment by tsolakoua** [comment]
I added most of the great questions that @estherrgarcia proposed in the document.

File: talent/sourcerers/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Questions to learn sourcerers better
+
+According to research, it becomes harder to establish a personal relationship with other team members when working remotely.
+
+In order to know you better, as a new member we would like you to reply a few have personal questions, as well as share with us your social media accounts you are comfortable with. By doing so we can establish more personal relationships outside of work activities.
+
+We have created a list of the questions and below are the instructions on how to successfully respond to the questions and make other members read them.
```
Comment (dpordomingo) [inline talent/sourcerers/README.md]: should be answered **all** the questions?
I think if someone does not want to answer some of them, she/he should feel free to remove it from its questionnaire.

If so, I'd explain it in this README.

**Pull request merged**